templates/admin.php

```diff
@@ -64,6 +64,9 @@
         </table>
         <input class="submit-button" id="auth_submit" type="submit" value="<?php p($l->t('Save'));?>" />
         <span id="auth_save_msg"></span>
+        <div class="warning">
+            <strong>Caution:</strong> The use of the <em>"Scan only this"</em> and <em>"Do not scan this"</em> settings should be approached with caution. Using these settings allows malicious users to upload and distribute malicious content via the Nextcloud instance. It is recommended that you carefully consider the implications of these settings and use them in a way that does not jeopardize the security of your system and data.
```

Comment:
The sentence "Using these settings allows malicious users to upload and distribute malicious content via the Nextcloud instance." feels a bit off, because this is also possible without any antivirus running in Nextcloud, but this sentence suggests that the settings will open this security risk.

Reply:
I also would recommend here to add examples, what mistakes can be made by using the settings